neofetch

```diff
@@ -1922,34 +1922,28 @@ get_ascii() {
         [[ "$ascii" != "distro" && ! -f "$ascii" ]] && \
             err "Ascii: Ascii file not found, using distro ascii."
 
-        # Lowercase the distro name so we can match it to a file.
-        if (("$bash_version" <= 3)); then
-            ascii="$(tr '[:upper:]' '[:lower:]' <<< "$ascii_distro")"
-        else
-            ascii="${ascii_distro,,}"
-        fi
-
-        if [[ -f "/usr/share/neofetch/ascii/distro/${ascii/ *}" ]]; then
-            ascii="/usr/share/neofetch/ascii/distro/${ascii/ *}"
+        # Find ascii art dir.
+        if [[ -d "/usr/share/neofetch/ascii/distro" ]]; then
+            ascii_dir="/usr/share/neofetch/ascii/distro"
 
-        elif [[ -f "/usr/local/share/neofetch/ascii/distro/${ascii/ *}" ]]; then
-            ascii="/usr/local/share/neofetch/ascii/distro/${ascii/ *}"
+        elif [[ -d "/usr/local/share/neofetch/ascii/distro" ]]; then
+            ascii_dir="/usr/local/share/neofetch/ascii/distro"
 
-        elif [[ -f "/data/data/com.termux/files/usr/share/neofetch/ascii/distro/${ascii/ *}" ]]; then
-            ascii="/data/data/com.termux/files/usr/share/neofetch/ascii/distro/${ascii/ *}"
+        elif [[ -d "/data/data/com.termux/files/usr/share/neofetch/ascii/distro" ]]; then
+            ascii_dir="/data/data/com.termux/files/usr/share/neofetch/ascii/distro"
 
         else
             get_script_dir 2>/dev/null
+            ascii_dir="${script_dir}/ascii/distro"
+        fi
 
-            # If the ascii file doesn't exist fallback to text mode.
-            if [[ -f "$script_dir/ascii/distro/${ascii/ *}" ]]; then
-                ascii="$script_dir/ascii/distro/${ascii/ *}"
+        # Remove 'linux' from 'linux mint'.
+        [[ "$ascii_distro" =~ ^linux ]] && ascii_distro="${ascii_distro/linux }"
 
-            else
-                to_off "Ascii: Ascii file not found, falling back to text mode."
-                return
-            fi
-        fi
+        # ascii_dir/first_letter (example: 'ascii_dir/m'*)
+        for art in "${ascii_dir}/${ascii_distro:0:1}"*; do
```

Comment:
We loop over the entire ascii art folder here but it's optimized so that we only loop over files that match the first letter of `$ascii_distro`.

For example:

```sh
# If the distro is Arch Linux then the real command will look like this:
for art in bla/bla/ascii/distro/a*; do
```

The speed is the same as the current ascii handling this way.